Tests/CreateAPITests/Snapshots.swift

```diff
@@ -5,26 +5,28 @@
 import Foundation
 import XCTest
 
-// TODO: Find it automatically
-var projectPath = ("~/Developer/CreateAPI/" as NSString).expandingTildeInPath
+fileprivate let generateSnapshots = false
+fileprivate let openDiff = false
 
 func compare(expected: String, actual: String, file: StaticString = #file, line: UInt = #line) throws {
-    let env = ProcessInfo.processInfo.environment
     let expectedURL = Bundle.module.resourceURL!
         .appendingPathComponent("Expected")
         .appendingPathComponent(expected)
     let actualURL = URL(fileURLWithPath: actual)
-    if env["GENERATE_SNAPSHOTS"] == "true" || !FileManager.default.fileExists(atPath: expectedURL.path) {
-        let projectPath = (projectPath as NSString).expandingTildeInPath
-        let destinationURL = URL(fileURLWithPath: projectPath + "/Tests/CreateAPITests/Expected/\(actualURL.lastPathComponent)")
+    
+    if generateSnapshots || !FileManager.default.fileExists(atPath: expectedURL.path) {
+        let destinationURL = URL(fileURLWithPath: #filePath)
+            .deletingLastPathComponent()
+            .appendingPathComponent("Expected")
+            .appendingPathComponent(actualURL.lastPathComponent)
         try? FileManager.default.removeItem(at: destinationURL)
         try FileManager.default.copyItem(at: actualURL, to: destinationURL)
     } else {
-        try diff(expectedURL: expectedURL, actualURL: actualURL, file: file, line: line)
+        try diff(expectedURL: expectedURL, actualURL: actualURL, file: file)
     }
 }
 
-private func diff(expectedURL: URL, actualURL: URL, file: StaticString = #file, line: UInt = #line) throws {
+fileprivate func diff(expectedURL: URL, actualURL: URL, file: StaticString = #file) throws {
```

Comment:
What's the need to change these to `fileprivate` when `private` does the trick? 

https://realm.github.io/SwiftLint/private_over_fileprivate.html

Reply:
I guess just a personal preference, where `fileprivate` is more declarative that it's, well, private to the file. Doesn't really matter to me though.